packages/calypso-e2e/src/lib/blocks/block-flows/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './instagram';
 export * from './twitter';
+export * from './pay-with-paypal';
```

Comment:
I'm good with the directory changes you made, but I was initially a little worried about the bloating of this `block-flows` directory given how many blocks we might end up with for our smoke tests.

Since our specs are divided up by the blocks source (e.g. jetpack vs ETK vs core), what do you think about keeping that subdivision in the directories here? Too nested?

Reply:
Personally, I don't think it's necessary to divide up the block flows - just like with pages/components, it can live all together in the same directory. Since the names are unique too, I don't see potential for it to clash either.